### PR TITLE
Day1 and Day2 assignment

### PR DESCRIPTION
**Pull request by mayuriswain33**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Update on Day1 and Day2 assignment

Test URLs:
- Before: https://main--eds-website--mayuriswain33.hlx.live/
- After: https://mayuri-develop--eds-website--mayuriswain33.hlx.live/

### CODE REVIEW

File: blocks/Video/Video.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
+.video {
```
Comment (noormd08) [inline blocks/Video/Video.css:1]: Keep the folder & file name as smaller case.

File: blocks/banner/banner.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+background-color:#0000FF;
+color:#fff;
+font-size:25px;
+position: relative;
+padding: 10px;
+}
+.banner img{
+width: 100%;
+height: 300px;
+}
```
Comment (noormd08) [inline blocks/banner/banner.css]: Add proper indentation.

File: blocks/cards/cards.css [hunk 3]
```diff
@@ -25,3 +25,28 @@
   aspect-ratio: 4 / 3;
   object-fit: cover;
 }
+
+/* Day 2 – Part 1  Assignment #5 start  */
+.anchor-anchorLink {
```
Comment (noormd08) [inline blocks/cards/cards.css]: Do not use camelcase for the CSS classname. Always go with hyphen separator.

File: blocks/cards/cards.css [hunk 4]
```diff
@@ -25,3 +25,28 @@
...
+  text-decoration: none; 
+  color: #333; 
+  font-size: 16px; 
+  width: 200px;
+  margin-left: 25px;
+}
+
+.anchor-anchorLink:hover {
+  background-color: #e0e0e0; 
+  color: #000;
```
Comment (noormd08) [inline blocks/cards/cards.css]: Try to use the color variables instead of hardcoding here.

File: styles/styles.css [hunk 5]
```diff
@@ -255,3 +255,49 @@ main .section.highlight {
...
+  font-size: 16px;          
+  display: inline-block;     
+}
+
+.section.linkstyle .button-container .button:hover {
+  background-image: linear-gradient(rgba(98, 189, 219, 0.84), rgba(65, 156, 199, 0.84) 50%);
+  color:#000;
+}
+
+/* Day 2 – Part 1  Assignment #1 end  */
```
Comment (noormd08) [inline styles/styles.css:289]: Keep the clean CSS for generic styles, make sure it is reusable.

**Comment by mayuriswain33** [comment]
Noor, I did all the required changes, please check.

File: blocks/Video/Video.css (hunk 1 above)
Comment (mayuriswain33) [reply blocks/Video/Video.css:1]: I have changed in my visual studio in the next commit it will be change.

File: blocks/banner/banner.css (hunk 2 above)
Comment (mayuriswain33) [reply blocks/banner/banner.css]: I have changed the CSS.

File: blocks/cards/cards.css (hunk 3 above)
Comment (mayuriswain33) [reply blocks/cards/cards.css]: I have modified the classname.

File: blocks/cards/cards.css (hunk 4 above)
Comment (mayuriswain33) [reply blocks/cards/cards.css]: I have changed the CSS.

File: styles/styles.css (hunk 5 above)
Comment (mayuriswain33) [reply styles/styles.css:289]: CSS changed.

File: styles/styles.css [hunk 6]
```diff
@@ -249,9 +254,42 @@ main > .section:first-of-type {
...
+.light-grey{
+  background-color: var( --homepage-background-color);
+  height: 400px;
+  color: var(--link-color);
+  font-size: 24px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center; 
+}
```
Comment (noormd08) [inline styles/styles.css:271]: Class name and properties are not in sync. Create a different name instead of light-grey

File: styles/styles.css [hunk 7]
```diff
@@ -249,9 +254,42 @@ main > .section:first-of-type {
...
+
+/* Day 1 (Part 2) Assignment #3 end */
+
+
+/* Day 2 – Part 1  Assignment #1 start  */
+
+.imagesection img {
```
Comment (noormd08) [inline styles/styles.css]: Use hyphen separator

File: blocks/Video/Video.css (hunk 1 above)
Comment (noormd08) [reply blocks/Video/Video.css:1]: Please push the changes.

File: styles/styles.css (hunk 6 above)
Comment (mayuriswain33) [reply styles/styles.css:271]: I have changed the CSS and classname.

File: styles/styles.css (hunk 7 above)
Comment (mayuriswain33) [reply styles/styles.css]: I have used hyphen separator.

**Pull request merged**